README.rst

```diff
@@ -5,15 +5,16 @@ libmc
 |status| |pypiv| |pyversions| |wheel| |license|
 
 libmc is a memcached client library for Python without any other
-dependencies in runtime. It's mainly written in C++ and Cython. libmc
-can be considered as a drop in replacement for libmemcached and
+dependencies at runtime. It's mainly written in C++ and Cython and
+can be considered a drop in replacement for libmemcached and
 `python-libmemcached <https://github.com/douban/python-libmemcached>`__.
 
-libmc is developing and maintaining by Douban Inc. Currently, It is
-working in production environment, powering all web traffics in
-douban.com. Realtime `benchmark
-result <https://travis-ci.org/douban/libmc/builds/57124335#L1611>`__ is
-available on travis.
+libmc is developed and maintained by Douban Inc. Currently, it is
+working in a production environment, powering all web traffic on
+`douban.com <https://www.semrush.com/website/douban.com/overview/>`__. [#]_
```

Comment:
I think it would be better to link to https://douban.com .
If rank by semrush is considered useful, maybe putting it inside parentheses or moving to comment area would be better.